internal/service/privatelinkendpoint/resource_privatelink_endpoint_migration_test.go

```diff
@@ -0,0 +1,51 @@
+package privatelinkendpoint_test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/mig"
+)
+
+func TestAccMigrationPrivateLinkEndpoint_basic(t *testing.T) {
+	var (
+		resourceName = "mongodbatlas_privatelink_endpoint.test"
+		orgID        = os.Getenv("MONGODB_ATLAS_ORG_ID")
+		projectName  = acctest.RandomWithPrefix("test-acc")
+		region       = "us-east-1"
+		providerName = "AWS"
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acc.PreCheckBasic(t) },
```

Comment:
```suggestion
		PreCheck:     func() { mig.PreCheckBasic(t) },
```
to also check that env var MONGODB_ATLAS_LAST_VERSION exists